src/PageCharacter/CharacterDisplay/Tabs/TabOptimize/Components/BuildDisplayItem.tsx

```diff
@@ -1,6 +1,6 @@
 import { Checkroom, ChevronRight } from '@mui/icons-material';
 import { Button, CardContent, Grid, Skeleton, Typography } from '@mui/material';
```

Comment:
Can you import `Box` here instead of on its own line as well, ty :) Same goes for other files

Reply:
fixed